sspl_test/functional_tests/ipmi_simulator/ipmisimtool

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```

Comment:
hope this is copied as it is, we dont need any changes here

Reply:
we need path changes here.